scripts/download_data.py

```diff
@@ -500,11 +512,13 @@ def parseArgs():
 @utils.Timed()
 def main(args):
     downloaderClass = sources[args.source]
-    downloader = downloaderClass(args)
-    downloader.downloadVcfs()
-    downloader.downloadBams()
-    downloader.downloadFastas()
-    downloader.cleanup()
+    try:
+        downloader = downloaderClass(args)
+        downloader.downloadVcfs()
+        downloader.downloadBams()
+        downloader.downloadFastas()
+    finally:
+        downloader.cleanup()
```

Comment:
Hmm, realised that this means that we always delete the checkpoint files too. Hopefully we're not going to need them any more now that htslib's networking code has been rewritten. Thoughts @dcolligan?